### ISSUE TITLE
Update documentation repo w/ DF, VE, and other information from the blogpost

### ISSUE THREAD

**Issue by idiom-bytes** [oceanprotocol/df-issues#101]
https://github.com/oceanprotocol/docs/issues/1133

**Comment by idiom-bytes** [comment]
Some feedback on certain subjects:
```
It’s a bit technical, but not hard to read. 3 things:
It’s not clear the distinction between active vs. passive rewards. I do understand them, but just going through the blog post, I don’t get the distinction.
Users will be able to claim rewards, and rewards will be in Ocean (weekly or whenever they claim) but it’s not well suggested that rewards are in Ocean, and that once you claim they go to your wallet.
Perhaps also a sort of visual/diagram, and how it all works. There are a lot of details, so having a value/mechanism flow, would be useful. How vault works, release periods, etc… Show don’t tell.
```
```
We should move this information into the docs. For longevity, it would be best to send people to the Docs rather than the blog post. You get a bit more structure as well? I’m saying that this should be in docs.ocean.com, this will be the most natural place for people to go. Blog post is great for getting initial attention, but in 1-2 months it will be more likely for them to go to the docs rather than medium.
```

**Comment by idiom-bytes** [comment]
> We should probably move all of the blog posts & intuitions from what we’re explaining into the ocean docs. Side-by-side w/ other products (marketplace, ocean.py) it makes sense to give it sufficiently prominence.

Yes & no. For the Nile team, let's be specific with sections/intuition to improve.
Yes => Move some high-level concepts from blog posts & elsewhere into documentation.
No => Create tutorial/blog on how people can leverage veOCEAN/Data Farming/etc...

Elsewhere (i.e. RealDataWhale wants to build tutorials) let's provide intuition and ask questions.

**Comment by idiom-bytes** [comment]
Break Rewards page into different pages/sections

veOCEAN
- How veCRV works, to lock, linear decay, withdrawing, can earn passive rewards, etc..

Data Farming
- How allocation works, how epochs work, how to use the FE
- simple farming tips (look for gaps in consume vs. allocated)
- live data vs. epoch data

Rewards
- reward function, how it's calculated, more intuition (blog posts)
- passive rewards, how does it work, how it's distributed etc...
- active rewards, how does it work, how it's distributed etc...

**Comment by idiom-bytes** [comment]
Perhaps document somewhere, all the actions that can be taken, and the number of tx's associated with them.

Creating a Lock = 2 tx
Updating Lock Time = 1 tx
Updating Lock Amount + Time = 2 tx
Withdrawing after Unlock = 1 tx
Managing your allocations = 1 tx
Claiming passive rewards = 1 tx
Claiming active rewards = 1 tx

**Comment by nickscavuzzo33** [comment]
https://docs.lens.xyz/docs/quick-setup
